package Ndnfs;

message DirInfo
{
  required string path = 1;
  required int32 type = 2;
}

message DirInfoArray
{
  repeated DirInfo di = 1;
}


